### PR TITLE
removing totalNumberOfBidsInSlot from Coordinator view

### PR DESCRIPTION
**Pull request by mmilenkovic**

### CODE REVIEW

File: src/views/coordinator/coordinator.view.jsx [hunk 1]
```diff
@@ -180,7 +180,6 @@ function Coordinator ({
                     >
                       <BidsList
```
Comment (elias-garcia) [inline src/views/coordinator/coordinator.view.jsx:181]: I think that we also need to remove the `totalNumberOfBidsInSlot` prop from the `BidsList` component right?

Comment (mmilenkovic) [reply src/views/coordinator/coordinator.view.jsx:181]: Nope, because it is being used in slot view.

Comment (elias-garcia) [reply src/views/coordinator/coordinator.view.jsx:181]: Ahh okay! And we don't need to perform any kind of check when we are not passing down this prop in the slot view?

Comment (mmilenkovic) [reply src/views/coordinator/coordinator.view.jsx:181]: I don't think so.

**Review by elias-garcia — APPROVED** [review]

**Pull request merged**